# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../e2fsprogs/26-linux-compile-h-usage.patch
# Copyright (C) 2004 - 2005 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---
--- ./misc/filefrag.c.orig	2004-06-26 13:03:48.820773848 +0200
+++ ./misc/filefrag.c	2004-06-26 13:04:05.359259616 +0200
@@ -37,6 +37,7 @@
 #include <sys/stat.h>
 #include <sys/vfs.h>
 #include <sys/ioctl.h>
+#include <linux/compiler.h>
 #include <linux/fd.h>
 
 int verbose = 0;
--- ./lib/ext2fs/getsectsize.c.orig	2004-06-26 13:00:49.114093416 +0200
+++ ./lib/ext2fs/getsectsize.c	2004-06-26 13:01:01.369230352 +0200
@@ -23,6 +23,7 @@
 #include <fcntl.h>
 #ifdef HAVE_LINUX_FD_H
 #include <sys/ioctl.h>
+#include <linux/compiler.h>
 #include <linux/fd.h>
 #endif
 
--- ./lib/ext2fs/getsize.c.orig	2004-10-08 12:45:23.000000000 -0400
+++ ./lib/ext2fs/getsize.c	2005-02-08 22:05:21.589995753 -0300
@@ -27,6 +27,7 @@
 #include <sys/ioctl.h>
 #endif
 #ifdef HAVE_LINUX_FD_H
+#include <linux/compiler.h>
 #include <linux/fd.h>
 #endif
 #ifdef HAVE_SYS_DISKLABEL_H
--- ./lib/ext2fs/ismounted.c.orig	2004-06-26 13:01:16.740893504 +0200
+++ ./lib/ext2fs/ismounted.c	2004-06-26 13:01:28.160157512 +0200
@@ -18,6 +18,7 @@
 #endif
 #include <fcntl.h>
 #ifdef HAVE_LINUX_FD_H
+#include <linux/compiler.h>
 #include <linux/fd.h>
 #endif
 #ifdef HAVE_MNTENT_H
--- ./lib/blkid/getsize.c.orig	2005-01-17 16:18:46.000000000 -0300
+++ ./lib/blkid/getsize.c	2005-02-08 22:04:57.704831686 -0300
@@ -27,6 +27,7 @@
 #include <sys/ioctl.h>
 #endif
 #ifdef HAVE_LINUX_FD_H
+#include <linux/compiler.h>
 #include <linux/fd.h>
 #endif
 #ifdef HAVE_SYS_DISKLABEL_H
